sig-list.md

```diff
@@ -27,7 +27,7 @@ When the need arises, a [new SIG can be created](sig-creation-procedure.md)
 |[Architecture](sig-architecture/README.md)|architecture|* [Brian Grant](https://github.com/bgrant0607), Google<br>* [Jaice Singer DuMars](https://github.com/jdumars), Google<br>* [Matt Farina](https://github.com/mattfarina), Samsung SDS<br>|* [Slack](https://kubernetes.slack.com/messages/sig-architecture)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-architecture)|* Regular SIG Meeting: [Thursdays at 19:00 UTC (weekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
 |[Auth](sig-auth/README.md)|auth|* [Mike Danese](https://github.com/mikedanese), Google<br>* [Mo Khan](https://github.com/enj), Red Hat<br>* [Tim Allclair](https://github.com/tallclair), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-auth)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-auth)|* Regular SIG Meeting: [Wednesdays at 11:00 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
 |[Autoscaling](sig-autoscaling/README.md)|autoscaling|* [Marcin Wielgus](https://github.com/mwielgus), Google<br>|* [Slack](https://kubernetes.slack.com/messages/sig-autoscaling)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-autoscaling)|* Regular SIG Meeting: [Mondays at 14:00 UTC (biweekly/triweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
-|[AWS](sig-aws/README.md)|aws|* [Justin Santa Barbara](https://github.com/justinsb)<br>* [Kris Nova](https://github.com/kris-nova), Heptio<br>* [Nishi Davidson](https://github.com/d-nishi), AWS<br>|* [Slack](https://kubernetes.slack.com/messages/sig-aws)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-aws)|* Regular SIG Meeting: [Fridays at 9:00 PT (Pacific Time) (biweekly)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
+|[AWS](sig-aws/README.md)|aws|* [Justin Santa Barbara](https://github.com/justinsb)<br>* [Kris Nova](https://github.com/kris-nova), Heptio<br>* [Nishi Davidson](https://github.com/d-nishi), AWS<br>|* [Slack](https://kubernetes.slack.com/messages/sig-aws)<br>* [Mailing List](https://groups.google.com/forum/#!forum/kubernetes-sig-aws)|* Regular SIG Meeting: [Fridays at 9:00 PT (Pacific Time) (biweekly starting Friday January 11, 2019)](https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit)<br>
```

Comment:
Okay, I was noticing a few that are specifying a date which the "biweekly" part actually starts on, and some that will use it to define things like "- except fourth Tuesday every month". It's at least staying consistent with what other sigs have.

```bash
→ grep -i "frequency:" sigs.yaml                                                                                                               
      frequency: biweekly
      frequency: weekly
      frequency: weekly
      frequency: biweekly
      frequency: biweekly/triweekly
      frequency: biweekly starting Friday January 11, 2019
      frequency: biweekly
      frequency: biweekly
      frequency: biweekly
      frequency: biweekly
        frequency: weekly
      frequency: biweekly
      frequency: weekly
      frequency: weekly
      frequency: weekly
      frequency: weekly
      frequency: weekly
      frequency: biweekly
      frequency: biweekly
      frequency: biweekly
      frequency: weekly
      frequency: weekly - except fourth Tuesday every month
      frequency: monthly - fourth Wednesday every month
      frequency: biweekly
      frequency: biweekly
      frequency: biweekly
      frequency: biweekly
      frequency: weekly
      frequency: biweekly
      frequency: weekly
      frequency: biweekly
      frequency: biweekly
      frequency: biweekly
      frequency: bi-weekly
      frequency: biweekly starting Thursday June 7, 2018
      frequency: biweekly starting Thursday June 14, 2018
      frequency: weekly
      frequency: biweekly
      frequency: weekly
        frequency: bi-weekly
      frequency: weekly
      frequency: bi-weekly
      frequency: monthly - first Wednesday every month
      frequency: bi-weekly
      frequency: weekly
      frequency: biweekly (On demand)
      frequency: bi-weekly (On demand)
      frequency: bi-weekly
      frequency: bi-weekly
      frequency: weekly
      frequency: biweekly
      frequency: weekly
      frequency: biweekly
      frequency: bi-weekly
      frequency: weekly
      frequency: weekly
      frequency: bi-weekly
```

The 2019 is purely to cover incase we're not still aligned with biweekly cadence from 2015.

Reply:
I still think the calendar invite should be the source of truth here. If I'm interested in a SIG meeting, I join the SIG mailing list with the expectation that the calendar invite is up-to-date.

AFAIK, the only times people pop over to k/community for meeting info are:
- When they're not part of the SIG mailing list
- When the mailing list invites are missing / incomplete

Reply:
agree with @justaugustus - I think the calendar invite from the mailing list which is the source of truth for SIG AWS should be the preferred mechanism.

Reply:
I guess I'm opposite of @justaugustus.  I try to avoid subscribing to each SIG's mailing list when I'm trying to just drop by a single meeting.

I believe sigs.yaml to be the source of record ("what should be") while the calendar invites are source of truth ("what is").  A lack of automation to reconcile the two is the state of today, but that's not going to stop me from suggesting we reconcile the two when we notice they differ.

Reply:
I get the sensitivity to the optics of "sig aws just started as of 2019" but I think you could make the case that the history of this repo will prove otherwise.

Reply:
I suggested the format: “2019 start date: Jan 11th” That won’t be confusing and achieves the clarity you request. Sounds good?

Reply:
works for me